diff --git a/node_modules/chrome-devtools-frontend/front_end/timeline_model/TimelineModel.js b/node_modules/chrome-devtools-frontend/front_end/timeline_model/TimelineModel.js
index 573e8e4..c614346 100644
--- a/node_modules/chrome-devtools-frontend/front_end/timeline_model/TimelineModel.js
+++ b/node_modules/chrome-devtools-frontend/front_end/timeline_model/TimelineModel.js
@@ -264,9 +264,9 @@ TimelineModel.TimelineModel = class {
       workers: workersDevToolsMetadataEvents.filter(checkSessionId).sort(SDK.TracingModel.Event.compareStartTime)
     };
     if (mismatchingIds.size) {
-      Common.console.error(
+      /*Common.console.error(
           'Timeline recording was started in more than one page simultaneously. Session id mismatch: ' +
-          this._sessionId + ' and ' + mismatchingIds.valuesArray() + '.');
+          this._sessionId + ' and ' + mismatchingIds.valuesArray() + '.');*/
     }
     return result;
   }
